scenarios/kubernetes_e2e_test.py

```diff
@@ -105,6 +108,16 @@ def test_docker(self):
         for call in self.callstack:
             self.assertTrue(call.startswith('docker'))
 
+    def test_default_tag(self):
+        """Ensure the default tag exists on gcr.io."""
+        args = self.parser.parse_args()
+        image = kubernetes_e2e.kubekins(args.tag)
+        m = re.match('gcr.io/([^:]+):(.+)', kubernetes_e2e.kubekins(args.tag))
+        self.assertIsNotNone(m)
```

Comment:
https://docs.python.org/2/library/unittest.html says
> This method should also be avoided when more specific methods are available (e.g. assertEqual(a, b) instead of assertTrue(a == b)), because they provide a better error message in case of failure.